README.md

```diff
@@ -27,12 +27,17 @@ Or in your site config:
  * `mod_elasticsearch2.host`
  * `mod_elasticsearch2.port`
 
-Elastic search normally doesn't count hits beyond 10K. To enable or disable counting the real
-total number of hits set the config key:
+Config keys:
 
- * `mod_elasticsearch2.track_total_hits`
+ * `mod_elasticsearch2.track_total_hits` Elastic search normally doesn't count hits beyond 10K.
+   To enable or disable counting the real total number of hits set this config.
+   This config key defaults to `true`.
 
-This config key defaults to `true`.
+ * `mod_elasticsearch2.default_operator`  The default text operator for words in a query string.
+   This is either AND or OR.  Defaults to 'OR'.
+
+ * `mod_elasticsearch2.no_automatic_wildcard` Automatically add wildcards to the words in a text
+   search string. The rewrites `"The quick fox"` to `((the | the*) (quick | quick*) (fox | fox*)) | "the quick fox"`  Defaults to `true`.
```

Comment:
I guess the description for this config setting needs a negative (Prevents adding..), or..maybe it is better to have automatic wildcards off by default? To enable them explicitly?

Reply:
Fixed description.

In our experience with other sites it is good to have it 'on' per default.  Especially as in Dutch we have a lot of compound words.